<?xml version="1.0"?>

<launch>

 <param name="/use_sim_time" value="$(arg simulation)"/>

  <node name="lpid_velocity" pkg="ten_package" type="pid_velocity.py" output="screen">
      <rosparam param="Kp">400</rosparam>
      <rosparam param="Ki">100</rosparam>
      <rosparam param="Kd">0</rosparam>
      <rosparam param="out_min">-1023</rosparam>
      <rosparam param="out_max">1023</rosparam>
      <rosparam param="rate">30</rosparam>
      <rosparam param="timeout_ticks">4</rosparam>
      <rosparam param="rolling_pts">5</rosparam>

  </node>

  <node name="rpid_velocity" pkg="ten_package" type="pid_velocity.py" output="screen">
      <rosparam param="Kp">400</rosparam>
      <rosparam param="Ki">100</rosparam>
      <rosparam param="Kd">0</rosparam>
      <rosparam param="out_min">-1023</rosparam>
      <rosparam param="out_max">1023</rosparam>
      <rosparam param="rate">30</rosparam>
      <rosparam param="timeout_ticks">4</rosparam>
      <rosparam param="rolling_pts">5</rosparam>
 </node>

</launch>
